### ISSUE TITLE
About ACO_resnet34.ckpt and L1 error of steering prediction on NuScenes dataset

### ISSUE THREAD

**Issue by Yingxiu-Chang** [metadriverse/ACO#10]
Hi, 
I downloaded your ACO_resnet34.ckpt and try to test a single image to predict the steering value. 
You had mentioned in your manuscript that the L1 error of steering prediction on NuScenes dataset are normalized to [0, 1]. 
When I fed an image [1, 3, 224, 224] into the model as:
```
net = resnet34()
net.load_state_dict(torch.load('ACO_resnet34.ckpt'), strict=False) 
output = net(image)
```
The output is [1, 1000] which is not a steering value. 
Did I miss something or misunderstood?
Best 
Hugh
